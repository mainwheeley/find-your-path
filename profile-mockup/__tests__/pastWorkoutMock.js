import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import PastWorkouts from '../comps/components/pastWorkouts';
import { shallow } from 'enzyme';
describe('Profile Bar', function() {
    it('renders correctly', () => {
	const tree = shallow(<PastWorkouts />)
	expect(tree).toMatchSnapshot();
    });
    it('length 1', function() {
	const tree = shallow(<PastWorkouts />)
	expect(tree.length).toBe(1);
    });
});
