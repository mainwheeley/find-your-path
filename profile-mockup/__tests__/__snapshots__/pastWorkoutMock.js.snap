// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Bar renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flex": 1,
        "justifyContent": "center",
        "margin": 10,
      }
    }
  >
    <RadioForm
      animation={true}
      buttonColor="#8BC34A"
      disabled={false}
      formHorizontal={true}
      initial={0}
      labelColor="#000"
      labelHorizontal={true}
      onPress={[Function]}
      radio_props={
        Array [
          Object {
            "label": "All ",
            "value": 0,
          },
          Object {
            "label": "Walk ",
            "value": 1,
          },
          Object {
            "label": "Run ",
            "value": 2,
          },
          Object {
            "label": "Bike ",
            "value": 3,
          },
          Object {
            "label": "Other ",
            "value": 4,
          },
        ]
      }
      selectedButtonColor="#2196f3"
      selectedLabelColor="#000"
      wrapStyle={Object {}}
    />
  </View>,
  "nodes": Array [
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "flex": 1,
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <RadioForm
        animation={true}
        buttonColor="#8BC34A"
        disabled={false}
        formHorizontal={true}
        initial={0}
        labelColor="#000"
        labelHorizontal={true}
        onPress={[Function]}
        radio_props={
          Array [
            Object {
              "label": "All ",
              "value": 0,
            },
            Object {
              "label": "Walk ",
              "value": 1,
            },
            Object {
              "label": "Run ",
              "value": 2,
            },
            Object {
              "label": "Bike ",
              "value": 3,
            },
            Object {
              "label": "Other ",
              "value": 4,
            },
          ]
        }
        selectedButtonColor="#2196f3"
        selectedLabelColor="#000"
        wrapStyle={Object {}}
      />
    </View>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <PastWorkouts />,
    "_instance": PastWorkouts {
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {
        "viewType": 0,
      },
      "updater": Updater {
        "_renderer": [Circular],
      },
      "workoutSeparator": [Function],
    },
    "_newState": null,
    "_rendered": <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "flex": 1,
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <RadioForm
        animation={true}
        buttonColor="#8BC34A"
        disabled={false}
        formHorizontal={true}
        initial={0}
        labelColor="#000"
        labelHorizontal={true}
        onPress={[Function]}
        radio_props={
          Array [
            Object {
              "label": "All ",
              "value": 0,
            },
            Object {
              "label": "Walk ",
              "value": 1,
            },
            Object {
              "label": "Run ",
              "value": 2,
            },
            Object {
              "label": "Bike ",
              "value": 3,
            },
            Object {
              "label": "Other ",
              "value": 4,
            },
          ]
        }
        selectedButtonColor="#2196f3"
        selectedLabelColor="#000"
        wrapStyle={Object {}}
      />
    </View>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <PastWorkouts />,
}
`;
